website/docs/d/schema_template_service_graph.html.markdown

```diff
@@ -24,15 +24,15 @@ data "mso_schema_template_service_graph" "example" {
 
 ## Argument Reference ##
 
-* `schema_id` - (Required) The schema ID of the Service Graph.
-* `template_name` - (Required) The template name of the Service Graph.
+* `schema_id` - (Required) The schema ID under which you want to deploy Service Graph.
+* `template_name` - (Required) The template name under which you want to deploy Service Graph.
 * `service_graph_name` - (Required) The name of the Service Graph.
 
 ## Attribute Reference ##
 
-* `site_nodes` - (Read-Only) A list of site nodes for the Service Graph.
-    * `node_index` - (Read-Only) The index of the Service Node.
-    * `service_node_type` - (Read-Only) The type of the Service Node.
-    * `site_id` - (Read-Only) The site ID of the Service Node.
-    * `tenant_name` - (Read-Only) The tenant name of the Service Node.
-    * `node_name` - (Read-Only) The name of the site level Service Node.
+* `service_node_type` - (Read-Only) **Deprecated**. The type of the Service Node.
+* `service_node` - (Read-Only) List of service nodes attached to Service Graph.
+    * `service_node.type` - (Read-Only) Type of Service Node attached to the Service Graph.
```

Comment:
```suggestion
    * `type` - (Read-Only) Type of Service Node attached to the Service Graph.
```

Reply:
`description` - attribute is missing.

Reply:
added.

Reply:
Please remove the service_node in-front of the `type`.